buffer/src/main/java/io/netty/buffer/PoolArena.java

```diff
@@ -321,15 +321,10 @@ void freeChunk(PoolChunk<T> chunk, long handle, SizeClass sizeClass, ByteBuffer
         int tableIdx;
         PoolSubpage<T>[] table;
         if (isTiny(elemSize)) { // < 512
-            tableIdx = elemSize >>> 4;
+            tableIdx = tinyIdx(elemSize);
```

Comment:
@franz1981 guess these methods will be inlined?

Reply:
yeah these are static and super hot so it should be inlined